Composer/packages/server/src/locales/en-US.json

```diff
@@ -227,6 +233,15 @@
   "any_string_f22dc2e1": {
     "message": "any string"
   },
+<<<<<<< HEAD
```

Comment:
remove merge conflict marker. Also, this PR shouldn't change this file. Can revert to the version on the bot-project feature branch